tools/editor/property_editor.h

```diff
@@ -219,6 +219,8 @@ class PropertyEditor : public Control {
 	void _edit_button(Object *p_item, int p_column, int p_button);
 
 	void _node_removed(Node *p_node);
+
+friend class ProjectExportDialog;
```

Comment:
need to add `friend` to set property value automatically after creating keystore